static/js/message_edit.js

```diff
@@ -854,4 +854,27 @@ exports.handle_narrow_deactivated = function () {
     }
 };
 
+exports.move_topic_containing_message_to_stream =
+    function (message_id, new_stream_id, new_topic_name) {
+        const request = {
+            stream_id: new_stream_id,
+            propagate_mode: 'change_all',
+            topic: new_topic_name,
+        };
+        channel.patch({
+            url: '/json/messages/' + message_id,
+            data: request,
+            success: function () {
+
```

Comment:
This should be just empty right? Since server is already taking care of the success case.

Reply:
Yeah; probably worth a comment :)